[CI] IndexCommitTimestampFieldRangeTests testFieldValueRangeForBatchedCompoundCommit failing

**Build Scans:**
- [elasticsearch-intake #39586 / part2](https://gradle-enterprise.elastic.co/s/x5pp6xqwrvypy)
- [elasticsearch-pull-request #145169 / part-2](https://gradle-enterprise.elastic.co/s/n3u2nx7qzdous)
- [elasticsearch-pull-request #145177 / part-2](https://gradle-enterprise.elastic.co/s/wwb2uko6kokrq)
- [elasticsearch-pull-request #145169 / part-2](https://gradle-enterprise.elastic.co/s/6s47wbcywrpak)
- [elasticsearch-pull-request #145112 / part-2](https://gradle-enterprise.elastic.co/s/uxbkq5ii73gai)
- [elasticsearch-pull-request #145112 / part-2](https://gradle-enterprise.elastic.co/s/jtbmrjw2k5qxi)
- [elasticsearch-pull-request #145110 / part-2](https://gradle-enterprise.elastic.co/s/5fb2vv736vkfq)
- [elasticsearch-pull-request #145110 / part-2](https://gradle-enterprise.elastic.co/s/k7abv3r6g7xvo)
- [elasticsearch-pull-request #145094 / part-2](https://gradle-enterprise.elastic.co/s/o64w5y2wam2lg)
- [elasticsearch-pull-request #145094 / part-2](https://gradle-enterprise.elastic.co/s/f7ufkhjjoj2m4)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:stateless:test" --tests "org.elasticsearch.xpack.stateless.commits.IndexCommitTimestampFieldRangeTests.testFieldValueRangeForBatchedCompoundCommit" -Dtests.seed=D27C22DE5DC6130B -Dtests.locale=gaa-GH -Dtests.timezone=Australia/ACT -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.stateless.commits.IndexCommitTimestampFieldRangeTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testFieldValueRangeForBatchedCompoundCommit'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: null
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testFieldValueRangeForBatchedCompoundCommit
- [main] 1 consecutive failures in step part2
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 18 failures in test testFieldValueRangeForBatchedCompoundCommit (5.5% fail rate in 326 executions)
- [main] 13 failures in step part-2 (6.4% fail rate in 203 executions)
- [main] 2 failures in step part-2-fips (66.7% fail rate in 3 executions)
- [main] 2 failures in step part-2-fips-140-3 (66.7% fail rate in 3 executions)
- [main] 8 failures in pipeline elasticsearch-pull-request (4.0% fail rate in 200 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -473,9 +473,6 @@ tests:
 - class: org.elasticsearch.xpack.stateless.commits.StatelessFileDeletionIT
   method: testDeleteIndexWhileNodeStopping
   issue: https://github.com/elastic/elasticsearch/issues/148807
-- class: org.elasticsearch.xpack.stateless.commits.IndexCommitTimestampFieldRangeTests
-  method: testFieldValueRangeForBatchedCompoundCommit
-  issue: https://github.com/elastic/elasticsearch/issues/148961
 - class: org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT
   method: test {p0=mixed_cluster/90_ml_data_frame_analytics_crud/Start and stop old regression job}
   issue: https://github.com/elastic/elasticsearch/issues/147516

